checkers/ruleguard_checker.go

```diff
@@ -0,0 +1,96 @@
+package checkers
+
+import (
+	"bytes"
+	"go/ast"
+	"go/token"
+	"io/ioutil"
+	"log"
+
+	"github.com/go-lintpack/lintpack"
+	"github.com/quasilyte/go-ruleguard/ruleguard"
+)
+
+func init() {
+	var info lintpack.CheckerInfo
+	info.Name = "ruleguard"
+	info.Tags = []string{"style", "experimental"}
+	info.Params = lintpack.CheckerParams{
+		"rules": {
+			Value: "",
+			Usage: "path to a gorules file",
+		},
+	}
+	info.Summary = "Detects custom rules violations"
```

Comment:
Sounds depressive. `Allows users to define custom rules`, WDYT ?

Reply:
Fixed.